Link fixes + WR ticket (#2206)
cloudflare/cloudflare-docs: products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md

## Activate APO

> To create the connection between WordPress and Cloudflare, you'll create an API token from your Cloudflare dashboard and add it to WordPress. To setup APO on a subdomain, see [Subdomains and subdirectories](/reference/subdomain-subdirectories).

Reviewer: ```suggestion
To create the connection between WordPress and Cloudflare, you will create an API token from your Cloudflare dashboard and add it to WordPress. To set up APO on a subdomain, refer to [Subdomains and subdirectories](/reference/subdomain-subdirectories).
```
